### PR TITLE
fix: (CXSPA-8033) add aria hidden attribute to 'required' asterisks

### PR DESCRIPTION
**Pull request by StanislavSukhanov**
* closes https://jira.tools.sap/browse/CXSPA-8033

### CODE REVIEW

File: feature-libs/user/account/components/login-form/login-form.component.html [hunk 1]
```diff
@@ -70,12 +70,3 @@
...
 </form>
-
-<ng-template #requiredAsterisk>
-  <abbr
-    *cxFeature="'a11yRequiredAsterisks'"
-    class="text-decoration-none"
-    title="{{ 'common.required' | cxTranslate }}"
-    >*</abbr
-  >
-</ng-template>
```
Comment (Pio-Bar) [inline feature-libs/user/account/components/login-form/login-form.component.html]: Good on you for cleaning it up, somebody must have left it here while removing the asterisks from the login page. 🤷

**Review by Pio-Bar — CHANGES_REQUESTED** [review]
Let's just add the attribute to the `abbr`, otherwise you can enter the empty group in read mode. We need to make sure the information about the field being required is still included somewhere. 🤔

**Comment by StanislavSukhanov** [comment]
@Pio-Bar fixed. The 'required' is still being narrated. It's being picked up by an inputs required attribute.

**Review by Pio-Bar — DISMISSED** [review]

**Review by Zeyber — DISMISSED** [review]

**Comment by developpeurweb** [comment]
@StanislavSukhanov your solution works well on JAWS. Now, let's complete the auditor's full request, so please bring back the Login and Reset Password asterisk removed on CXSPA-7418 and **apply your solution there as well**. I have left a comment on that ticket explaining why we can't remove audited content.

**Comment by StanislavSukhanov** [comment]
@developpeurweb It's done. Please have a look on that once again 🙏

**Review by developpeurweb — APPROVED** [review]

**Pull request merged**